key/key.go

```diff
@@ -0,0 +1,23 @@
+package key
+
+const (
+	Email         = "email"
+	CompanyKey    = "company_key"
+	Latency       = "latency"
+	Method        = "method"
+	URI           = "uri"
+	Status        = "status"
+	UserAgent     = "user_agent"
+	APIVersion    = "api_version"
+	Payload       = "payload"
+	User          = "user"
+	CorrelationID = "correlation_id"
+)
+
+type ctxKey int
+
+const (
+	CtxEmail ctxKey = iota
```

Comment:
0 as key?

Reply:
Why not? It is only for Context keys

Reply:
It needs to be separate type (this is why I created new one) otherwise it linter will cry

Reply:
But they will be sent to DD, wrong?

Reply:
I decided to go with iota as this is easy to use and I saw usage like that before. They are not meant to be written by hand anyway.

Reply:
https://stackoverflow.com/a/40891417 <-- exact question I asked myself